scripts/cloud_data_plane_setup.sh

```diff
@@ -5,7 +5,7 @@ if [[ $(uname -s) == "Darwin" ]]; then
   READLINK="greadlink"
 fi
 
-HOME=$(dirname "$($READLINK -f "$0")")
+HOME=$($READLINK -f .)
```

Comment:
I'm afraid current working directory is not suitable, it may occur some permission problems.

Let's just use the `$HOME/.cloud` directory.

Reply:
OK.
But it is an issue when it is not 'home directory' user.

Reply:
updated